### DISCUSSION TITLE
V3 beta release

### DISCUSSION THREAD

**Discussion by SimonCropp** [Humanizr/Humanizer#1451] [General]
any objections to releasing a v3 beta1?

my plan is to pint to the v3 milestone as release notes https://github.com/Humanizr/Humanizer/issues?q=is%3Aclosed+milestone%3Av3.0

@hazzik @MehdiK

**Comment by SimonCropp** [comment]
@hangy

**Reply by hangy** [reply]
I have no objections

**Comment by clairernovotny** [comment]
We need to fix versioning and signing first.
